src/librustdoc/html/render/mod.rs

```diff
@@ -1066,7 +1066,11 @@ themePicker.onblur = handleThemeButtonsBlur;
                 krates
                     .iter()
                     .map(|s| {
-                        format!("<li><a href=\"{}index.html\">{}</li>", ensure_trailing_slash(s), s)
+                        format!(
+                            "<li><a class=\"mod\" href=\"{}index.html\">{}</a></li>",
```

Comment:
```suggestion
                            "<li><a class='mod' href='{}index.html'>{}</a></li>",
```

Reply:
We're using double quotes everywhere else, so for consistency, I think we should keep them.

Reply:
I saw that we also use single quotes for quite a lot of place too.

Reply:
179 single quotes and 230 double quotes (looking for `='` and "=\"`). I wonder how we ended up in this mess... We should clean that up.

Reply:
Why not just make all single quotes? We don't need to escape that way..

Reply:
We could change that to
```
<div onclick="func('clicked');">...</div>
```

Reply:
Or to:

```
<div onclick='func("clicked");'>...</div>
```

:stuck_out_tongue: